### PR TITLE
Add utility commands for app hosting.

### PR DESCRIPTION
**Pull request by taeold**
We haven't finalized the design of these commands, but they sure are handy for local testing.

### CODE REVIEW

**Comment by taeold** [comment]
Current dependencies on/for this PR:
* `master`
  * **PR #6609** <a href="https://app.graphite.dev/github/pr/firebase/firebase-tools/6609?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
    * **PR #6610** <a href="https://app.graphite.dev/github/pr/firebase/firebase-tools/6610?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
      * **PR #6611** <a href="https://app.graphite.dev/github/pr/firebase/firebase-tools/6611?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
        * **PR #6612** <a href="https://app.graphite.dev/github/pr/firebase/firebase-tools/6612?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
          * **PR #6616** <a href="https://app.graphite.dev/github/pr/firebase/firebase-tools/6616?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/firebase/firebase-tools/6616?utm_source=stack-comment).

File: src/commands/apphosting-builds-create.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import * as apphosting from "../gcp/apphosting";
+import { logger } from "../logger";
+import { Command } from "../command";
+import { Options } from "../options";
+import { needProjectId } from "../projectUtils";
+
+function generateId(n = 6): string {
```
Comment (Yuangwang) [inline src/commands/apphosting-builds-create.ts:7]: Can this be moved out into a util file? I see you've written it 3 times now.

**Review by Yuangwang — APPROVED** [review]
Added one comment, good to merge after that is addressed

File: src/commands/apphosting-builds-create.ts (hunk 1 above)
Comment (taeold) [reply src/commands/apphosting-builds-create.ts]: Done!

**Comment by taeold** [comment]
### Merge activity

* **Dec 14, 3:44 PM**: [Graphite](https://app.graphite.dev/github/pr/firebase/firebase-tools/6616) rebased this pull request as part of a merge.

**Pull request merged**